egui_plot/src/items/mod.rs

```diff
@@ -223,12 +224,12 @@ impl PlotItem for HLine {
 
         // Round to minimize aliasing:
```

Comment:
No need anymore; epaint does this for us now

Reply:
nice!